aws/cloudfront/cloudfront.tf

```diff
@@ -1,66 +1,231 @@
 resource "aws_cloudfront_distribution" "cloudfront_distribution" {
```

Comment:
Maybe some comments/description of top of file to describe what the this "Module" does